src/app/dashboard/dashboard.component.ts

```diff
@@ -7,8 +7,10 @@ import { Competency } from '../../entity/competency';
 import { Lifecycles } from '../../entity/lifecycles';
 import { Search } from '../../entity/search';
 import { sleep } from '../shared/functions/loading';
-import { BuilderService } from '../core/builder/builder.service';
-import { CompetencyBuilder } from '../core/builder/competency-builder.class';
+import { BuilderService } from '../core/builder.service';
+//'../core/builder/builder.service'
+import { CompetencyBuilder } from 'src/entity/builder.class';
```

Comment:
```suggestion
import { CompetencyBuilder } from 'src/entity/builder.class';
```
This needs to be a relative import like '../../..'